.github/script-missing-mode-enforcement.py

```diff
@@ -0,0 +1,62 @@
+"""
+Script to scan the OpenTTD's script API for functions that miss checks for the
+function being called from the right mode (deity or company mode).
+
+When a function calls either ScriptObject::Command or ScriptObject::GetCompany
+then the function is considered dangerous. When one of the mode enforcement
+macros from script_error.hpp, i.e. EnforceDeityMode, EnforceCompanyModeValid or
+EnforceDeityOrCompanyModeValid, are called in the function, then we consider
+that the function has mode enforcement.
+
+Any dangerous function for which no enforcement is found are emitted as errors.
+"""
+
+import glob
+import re
+import sys
+
+def checkModeEnforcement(path):
+	errors = []
```

Comment:
It is really uncommon for Python scripts to use tabs; although fully allowed, it is uncommon.

The other scripts for OpenTTD we have been writing with the more default "4 spaces". Maybe good to follow that trend here too?